cita-network/src/node_manager.rs

```diff
@@ -950,3 +990,42 @@ impl ConnectedSelfReq {
         service.dialing_node = None;
     }
 }
+
+pub struct ModifiedConfigPeersReq {
+    peers: Vec<String>,
+}
+
+impl ModifiedConfigPeersReq {
+    pub fn new(peers: Vec<String>) -> Self {
+        ModifiedConfigPeersReq { peers }
+    }
+
+    pub fn handle(self, service: &mut NodesManager) {
+        //if new config deleted some peer,disconnect and remove it from known addrs
```

Comment:
Please keep a space between the // and comment.

Reply:
done